pkgs/os-specific/linux/tuxedo-rs/default.nix

```diff
@@ -1,29 +1,43 @@
 { lib
 , fetchFromGitHub
 , rustPlatform
+, testers
+, tuxedo-rs
 }:
 let
```

Comment:
Instead of using the `let ... in` construction in `callPackage` derivations it is generally better to use `rec` and re-use the variables inside `buildRustPackage` or `mkDerivation`. This allows to call `overrideAttrs` on the derivation and achieve the desired results.

Reply:
TIL... 🙏